python/pyproject.toml

```diff
@@ -17,6 +17,7 @@ dependencies = [
 [project.optional-dependencies]
 plot = [
```

Comment:
I should probably put this in a different group of optional dependencies. This would cause `pip install kaskada[plot]` to install both plotly and pydot. Maybe we want it a different `diagnose` or `internal` or `debug` group? Thoughts?

Reply:
Yeah, agreed. `diagnose` sounds good to me, unless we want to be explicit and do a `kaskada[plan]` group?

Reply:
Went with `explain` for the function and the optional dependency.